packages/runtime/client/rest/src/contract/endpoints/collection.ts

```diff
@@ -211,52 +236,217 @@ export interface CreateContractRequest {
   /**
    * The Marlowe Runtime utilizes the mandatory `changeAddress` and any additional addresses provided here
    * to search for UTxOs that can be used to balance the contract creation transaction.
+   *
    * @remarks
+   *
    * 1. When using single address wallets like Nami, it is not necesary to fill this field.
    * 2. If an address was provided in the `changeAddress` field, it is redundant to include it here (but it doesn't fail).
+   *
    * @see WalletAPI function {@link @marlowe.io/wallet!api.WalletAPI#getChangeAddress}
    * @see WalletAPI function {@link @marlowe.io/wallet!api.WalletAPI#getUsedAddresses}
    */
   usedAddresses?: AddressBech32[];
   /**
-   * TODO: Document
+   * UTxOs provided as collateral in case the Tx built will unexpectedly fail at the submit phase.
+   *
+   * <h4>Justification</h4>
+   * <p>
+   * The collateral mechanism is an important feature that has been designed to ensure
+   * successful smart contract execution.
+   *
+   * Collateral is used to guarantee that nodes are compensated for their work in case phase-2 validation fails.
+   * Thus, collateral is the monetary guarantee a user gives to assure that the contract has been carefully designed
+   * and thoroughly tested.
+   *
+   * Collateral amount is specified at the time of constructing the transaction.
+   * Not directly, but by adding collateral inputs to the transaction.
+   *
+   * The total balance in the UTXOs
+   * corresponding to these specially marked inputs is the transaction’s collateral amount.
+   *
+   * If the user fulfills the conditions of the guarantee, and a contract gets executed, the collateral is safe.
+   * </p>
+   * @see
+   * https://docs.cardano.org/smart-contracts/plutus/collateral-mechanism
    */
   collateralUTxOs?: TxOutRef[];
+
+  /**
+   * A Marlowe Contract or a Merkleized One (referred by its source Id) to create over Cardano
```

Comment:
Maybe "Merkleized One" should be "merkleized one" to not seem like it's a thing or a product.